wfe/web-front-end.go

```diff
@@ -227,7 +227,7 @@ func (wfe *WebFrontEndImpl) NewAuthorization(response http.ResponseWriter, reque
 
 	body, key, err := verifyPOST(request)
 	if err != nil {
-		wfe.sendError(response, "Unable to read/verify body", http.StatusBadRequest)
+		wfe.sendError(response, fmt.Sprintf("Unable to read/verify body: %v", err), http.StatusBadRequest)
```

Comment:
I'd like to minimize the degree to which we echo potentially unknown errors into client responses. I know other places don't necessarily get this right.

If the goal is more information for debugging, we can put the body into a wfe.log.Debug(). Or, if you've checked all the returned errors from verifyPOST and they don't echo user data or errors from deeper in the stack, then this is probably okay. I would also suggest %s / err.Value() rather than %v, just to be slightly more explicit about types.